engine/clients/weaviate/search.py

```diff
@@ -32,10 +32,10 @@ def init_client(cls, host, distance, connection_params: dict, search_params: dic
         cls.client = client
 
     @classmethod
-    def search_one(self, vector, meta_conditions, top) -> List[Tuple[int, float]]:
+    def search_one(self, query: Query, top: int) -> List[Tuple[int, float]]:
```

Comment:
classmethod uses `self` instead of `cls`

Reply:
Good catch.